### ISSUE TITLE
Misleading error message on ambiguous install command

### ISSUE THREAD

**Issue by kevinmichaelchen** [pkgxdev/pkgx#744]
Apologies for the newb question - I'm new to coreutils...

I want to install Golang globally:

```
tea +go.dev install go
```

However, this fails with:

```
install: missing destination file operand after 'go'
Try '/Users/kevin.c/.tea/gnu.org/coreutils/v9.4.0/bin/install --help' for more information.
```

Coreutils should already be pre-installed on macOS.

Any thoughts @jhheider?

**Comment by jhheider** [comment]
That's calling `/usr/bin/install go` in an environment with go available. So, `install` is complaining about missing second operand (`install --help`).

I believe you want `tea install go`. See https://docs.tea.xyz/using-tea/tea-install.

**Comment by jhheider** [comment]
Interesting. Seems like our errors aren't matching reality. How about `tea install go.dev`?

**Comment by kevinmichaelchen** [comment]
Doh 🙈 that worked!

**Comment by jhheider** [comment]
> ```
> tea install go
> 
> × multiple projects provide: go
> │ pls be more specific:
> │
> │     tea +encore.dev/go install go
> │     tea +go.dev install go
> │
> ╰─➤ https://help.tea.xyz/ambiguous-pkgspec
> ```

this is still an issue we should fix. in the `install` case, it should be providing `tea install ${pkg.name}`.

**Comment by mxcl** [comment]
You wanted `tea install go.dev`. `tea install` only works if you specify it precisely like that.

I agree this sucks, and will try to figure out what should be done about it.

This is ofc. exactly why I never wanted to add `tea install` and you should always be careful to give people what they claim they want…

**Comment by jhheider** [comment]
`tea install node` definitely works. I think the only (substantial) issue is that the error message shows incorrect usage on ambiguous names to the install keyword; it instead shows the ambiguous name error as if it was passed `tea +go install go`.

**Comment by felipecrs** [comment]
I would recommend you close this one, @jhheider, since the original problem was fixed.

**Comment by jhheider** [comment]
Maybe. I still consider the buggy error message to be an issue.

**Comment by felipecrs** [comment]
I see.

BTW this can also happen if user is not running v1 yet, and tries to issue a `tea install`.

**Comment by felipecrs** [comment]
No worries, it's perfectly understandable.

**Comment by mxcl** [comment]
Fixed by #815
